qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -202,7 +202,7 @@ def show_log_output(result_file):
     result_file = r"C:\driver_install.log"
     logger = logging.getLogger("driver_install")
     logger.setLevel(logging.DEBUG)
-    fh = logging.FileHandler(result_file)
+    fh = logging.FileHandler(result_file, mode="a+")

```

Comment:
@suqinhuang 
As we talked, "win_driver_install.py" is run in the guest and file "C:\driver_install.log" is created in guest,
seems can not handle it by case.

Any more comments for the PR?

Thanks.